libraries/chain/controller.cpp

```diff
@@ -662,6 +668,14 @@ struct controller_impl {
                                                                              majority_permission.id,
                                                                              active_producers_authority,
                                                                              conf.genesis.initial_timestamp );
+
+     if (conf.read_genesis) {
+        create_native_account(config::token_account_name, system_auth, system_auth);
```

Comment:
It seems that the creation of application accounts should be in the function of reading genesis data.

Reply:
Ok, I'll move them in next PR